src/zenml/cli/pipeline.py

```diff
@@ -73,6 +144,11 @@ def run_pipeline(python_file: str, config_path: str) -> None:
         python_file: Path to the python file that defines the pipeline.
         config_path: Path to configuration YAML file.
     """
+    # If the file was run with `python run.py, this would happen automatically,
+    #  In order to allow seamless switching between running directly and through
+    #  zenml, this is done at this point
+    sys.path.insert(0, os.path.abspath(os.path.dirname(python_file)))
```

Comment:
We could use the `source_utils.prepend_python_path` contextmanager here so we cleanup the python path afterwards. Not sure if that's ever going to be relevant (maybe in the tests?) as it's a CLI function which will exit right afterwards, but from experience it will come bite us at some point in the future :D

Reply:
I just checked, it is removed too early from the sys path when using source_utils.prepend_python_path

Reply:
Even if you wrap the entire rest of the function in it? I guess then just leave it as is

Reply:
I tried again, it worked, I must have done it wrong the first time